client/src/utils/PostUtils.js

```diff
@@ -58,3 +58,22 @@ export const getPostByID = async ( postID, setPost ) => {
             console.error("getPostByID error: ", error)
         })
 }
+
+// Provided a postID, deletes the post and updates the user's posts array state
+export const deletePost = async ( post, setUserPosts ) => {
```

Comment:
you don't have to do this change, but just as an fyi, with this type of logic where you have a chain of events occuring in the back-end (deleting the post, and then deleting the file), it's a better practice to handle all this logic from the back-end instead of making 2 separate calls from the front-end. This reduces room for errors when referencing these api endpoints